general-concepts/slotting/text.xml

```diff
@@ -60,8 +60,104 @@ automatically rebuilt</uri> when the subslot of a runtime dependency changes.
 </p>
 
 <p>
-For example, suppose package <c>foo</c> installs a library whose soname is
-different for different versions. It would be reasonable to use the soname version
+If an ebuild does not explicitly declare a sub-slot, the regular slot is used
+as the value of the sub-slot by default.
+</p>
+
+<p>
+You may wish to review the QA team's documentation on subslots on the
+<uri link="https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Subslots">
+wiki.</uri>
+</p>
+
+<note>
+Care must be taken when using sub-slots in a library ebuild for the first time.
+Adding sub-slots will trigger rebuilds for all the packages that already use sub-slot
+dependencies (e.g. Switching from SLOT="0" to SLOT="0/14" in <c>media-libs/libpng</c> and
+package <c>foo</c> depends on <c>libpng:0=</c>).
+Therefore, it's best if you start using sub-slots in the library when the existing library
+interface changes.
+</note>
+</body>
+
+<subsection>
+<title>ABI breakage</title>
+<body>
+
+<p>
+There are two ways a library can break compatibility with its consumers.
+</p>
+
+<ul>
+<li>
+ABI (Application Binary Interface): this affects binaries built before the change.
+Applications linked against a library pre-ABI break may not work correctly after
+the break. These changes are related to internal structure, such as the size of
+a <c>struct</c> or the type of an argument (e.g. integer width). Fixing this
+requires a <e>rebuild</e> of all consumers.
+</li>
+<li>
+API (Application Program Interface): this affects consumers at compile time and
+usually occurs when a library has deprecated and then removed a function. Fixing
+this requires a <e>code</e> change in consumers.
+</li>
+</ul>
```

Comment:
Please indent `<li>` and the text within.

Also, this looks like a candidate for `<dl>` instead of `<ul>` (I haven't checked its visual appearance though).

Reply:
Thanks!